src/utils/searchUtils.ts

```diff
@@ -254,8 +254,8 @@ export const filterPathways = (
         ...pathway.geography.map((s) => geographyLabel(s)),
         ...pathway.sectors.map((s) => s.name),
         ...pathway.metric,
-        pathway.publisher,
-        pathway.publicationYear,
+        pathway.publication.publisher.full,
```

Comment:
NB: Do we also want to be able to search on `publisher.short`? (IEA, instead of International Energy Agency)?

Reply:
Yeah, that's probably a good idea